lib/actions/google/drive/sheets/google_sheets.js

```diff
@@ -66,9 +76,15 @@ class GoogleSheetsAction extends google_drive_1.GoogleDriveAction {
                 }
                 catch (e) {
                     this.sanitizeGaxiosError(e);
-                    winston.error(`Failed execute for Google Sheets. Error: ${e.toString()}`, { webhookId: request.webhookId });
+                    let error = Hub.errorWith(http_errors_1.HTTP_ERROR.internal, `Failed execute for Google Sheets. Error: ${e.toString()}`);
```

Comment:
We've been creating `LOG_PREFIX` const instead of writing 'Google Sheets'

https://github.com/looker-open-source/actions/pull/601/files#diff-edbe3e2324dae5d23d86815669f573a8cfc8904a00b55c783ddc5ba583d9638eL17

Reply:
Added in the new commit